### ISSUE TITLE
Directive not working inside ng-view using angular route

### ISSUE THREAD

**Issue by tommykennedy** [rprovost/ng-backstretch#14]
Hi,

There seems to be an issue when using this directive within a partial html file using angular route. 

I have routes set up and they get called when links are clicked. The backstretch directive is on divs within these files but the images dont appear. 

Is there a scoping issue?

Regards
Tom

**Comment by rprovost** [comment]
@tommykennedy I'm actually using this in my own personal project inside a partial. I'll try to add an example to the demo to illustrate how it's been working for me.

**Comment by sebastientromp** [comment]
Would love to see this too. I've added the demo code to my own controller / view, the images are created into the backstretch div, but they aren't displayed

**Comment by smurugavel** [comment]
I agree with @tommykennedy and I can't get images to display with ng-view. Ryan, if you could a demo, that helps.

**Comment by aplongh** [comment]
rprovost, thanks for porting and sharing this awesome jquery plugin. I've had the same problem as others have mentioned, that is, the images do not show inside ng-view. An inspection of the rendered element shows this: `margin: 0px; padding: 0px; left: 0px; top: 0px; width: 1336px; height: 0px; overflow: hidden; position: absolute; z-index: -999998;` The relevant part is that height is set to 0. 

Did you have a chance to take a look at this?

Best regards, Ariel

**Comment by rprovost** [comment]
Hey All. Apologies for being MIA on this project for so long. I've been swamped with many other things lately. It looks like @jwanglof's PR #20 should have fixed the issue.

I've just pushed out a new release.
